apps/web-portal-client/lib/context/ApiContext.js

```diff
@@ -0,0 +1,5 @@
+import {createContext} from 'react'
+
+export const ApiContext = createContext({
```

Comment:
I'm thinking of renaming  `ApiContext` to `AppContext`, so it would contain not only 'api' but a general config object

Reply:
done